components/meta-packages/history/history

```diff
@@ -475,6 +475,8 @@ database/mariadb-55@5.5.55,5.11-2017.0.0.3
 database/mariadb-55/client@5.5.55,5.11-2017.0.0.3
 database/mariadb-55/library@5.5.55,5.11-2017.0.0.3
 database/mariadb-55/tests@5.5.55,5.11-2017.0.0.3
+database/mongodb-common@3.3.12-2018.0.0.3 database/mongodb-common@3.4.19-2018.0.0.0
+database/mongodb-33@3.3.12-2018.0.0.3 database/mongodb-34@3.4.19,5.11-2018.0.0.0
```

Comment:
Perhaps, we should just obsolete it? I'm not sure. In any case, replacing 3.3 with 3.4 on update will give system administrator a lot of work (as well as just removing it).

Reply:
Right, replacement may bring problems on update. Though, I still think unstable version should not be used, when stable stream is present. How is obsolescence done here? By keeping just the first part? `database/mongodb-common` replacement should be kept as is, I guess?

Reply:
You don't need mongodb-common replacement - it's no-op, package is not versioned.
To mark package obsolete keep just first part.

Reply:
Obsolete `{service/,}database/mongodb-33`, no replacement now.